internal/stage10.go

```diff
@@ -39,6 +39,7 @@ func testCd1(stageHarness *test_case_harness.TestCaseHarness) error {
 		ExpectedOutput: fmt.Sprintf(`cd: %s: No such file or directory`, directory),
 		FallbackPatterns: []*regexp.Regexp{
 			regexp.MustCompile(fmt.Sprintf(`^(can't cd to %s|((bash: )?cd: )?%s: No such file or directory)$`, directory, directory)),
+			regexp.MustCompile(`ash: cd: can't cd to ` + directory + `: No such file or directory`),
```

Comment:
```suggestion
			regexp.MustCompile(`^ash: cd: can't cd to ` + directory + `: No such file or directory$`),
```

Let's be more strict?